.github/workflows/ci.yml

```
@@ -32,7 +32,7 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: Use Node.js
-        uses: actions/setup-node@v2
+        uses: actions/setup-node@v3
         with:
           node-version: '14'
           cache: npm
```

```suggestion
        with:
          cache: npm
```
I guess these "node-version" inputs should be removed, as it now defaults to 16